### PR TITLE
Avoid use of "Ctrl-C" to detach from "riak attach" console

### LINKED ISSUE #197: Don't use "Ctrl-C" to detach from "riak attach" console [JIRA: TOOLS-237]
On some platforms (e.g. RHEL/CentOS), "Ctrl-C" immediately detaches the erlang process from the controlling terminal.

A better way to terminate the Eshell is to use the "Ctrl-G q" sequence, as advised by Eshell on startup.

### PR DESCRIPTION
**Pull request by kesslerm**
Fixes #197 (TOOLS-237)

### CODE REVIEW

File: priv/base/runner [hunk 1]
```diff
@@ -90,13 +90,13 @@ This is the primary script for controlling the $SCRIPT node.
         Attaches to the console of a $SCRIPT node running in the background
         using an Erlang remote shell, giving access to the Erlang shell and
         runtime messages. Prints "Node is not running!" when the node cannot
```
Comment (JeetKunDoug) [inline priv/base/runner]: "Node is not running!" is now replaced with "Node did not respond to ping!" per #199

**Comment by kesslerm** [comment]
Rebased on latest develop branch. Changed the help output to reflect the new message for ping failures.

File: priv/base/runner [hunk 2]
```diff
@@ -301,7 +301,7 @@ case "$1" in
         # Make sure a node is running
         node_up_check
 
-        echo "Remote Shell: Use \"Ctrl-C a\" to quit. q() or init:stop() will terminate the $SCRIPT node."
+        echo "Remote Shell: Use \"Ctrl-G q\" to quit."
```
Comment (kesslerm) [inline priv/base/runner:306]: @JeetKunDoug would like to keep the warning about q() or init:stop()

**Comment by kesslerm** [comment]
Added the warning about q() and init:stop() as a separate output line

**Comment by JeetKunDoug** [comment]
:+1: thanks for the updates.

**Pull request merged**